src/frontend/servers/y-provider/__tests__/getDocumentConnectionInfoHandler.test.ts

```diff
@@ -0,0 +1,174 @@
+import request from 'supertest';
+import { v4 as uuid } from 'uuid';
+
+const port = 5555;
```

Comment:
I can see the test fails, better to use another port, this one is used here:
https://github.com/suitenumerique/docs/blob/2e1b1121337af2ac305f04e6d0b2fc0f933956d9/src/frontend/servers/y-provider/__tests__/collaborationResetConnections.test.ts#L3

I guess the port of the previous test does not have the time to close.